.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+
+jobs:
+  linux-aarch64-wheels:
+    working_directory: ~/linux-aarch64-wheels
+    machine:
+      image: ubuntu-2004:2022.04.1
+    # resource_class is what tells CircleCI to use an ARM worker for native arm builds
+    # https://circleci.com/product/features/resource-classes/
+    resource_class: arm.medium
+    steps:
+      - checkout
+      - run:
+          name: Build the Linux aarch64 wheels.
+          environment:
+            CIBW_SKIP: "pp* *-musllinux_aarch64"
+            CIBW_BEFORE_BUILD_LINUX: bash packaging/install_dependencies.sh
```

Comment:
```suggestion
            CIBW_BEFORE_ALL_LINUX: yum install -y flex-devel bison-devel
```

Reply:
This was my initial idea, however, the version of flex (or bison?) in the repos for the `manylinux` image is too old for NMODL (disclaimer: I don't know why we require those specific minimum versions in the first place), so we need to install them from source.

Reply:
```suggestion
            CIBW_BEFORE_ALL_LINUX: bash packaging/install_dependencies.sh
```

Reply:
Ahhh right, good catch! I moved everything to `pyproject.toml` to keep the CI clutter-free (options can still be overridden with env variables), so I'll put this there as well.